### DISCUSSION TITLE
🐛 BUG: You are sending me too many emails. One per day would suffice.  Kind Regards,  Kayle Maclou

### DISCUSSION THREAD

**Discussion by kayle-maclou** [dailydotdev/daily#1581] [Feature Requests]
### What went wrong? 🤔

You are sending me too many emails. One per day would suffice.

Kind Regards,

Kayle Maclou

### Steps to Reproduce Issue

```bash
1. Go to '...'
2. Click on '....'
3. Scroll down to '....'
4. See the error
```

### Screenshots

```bash
![DESCRIPTION](LINK.png)
```

### Code of Conduct

- [X] I follow the conditions of this project.

**Comment by idoshamun** [comment]
Hi @kayle-maclou,

We're working on improving the email experience. I assume you refer to the emails about subscribed sources right?

**Comment by kayle-maclou** [comment]
Hi Ido,

It's unacceptable to receive this many emails in one day. Please
consolidate them into a single daily email.

Kind Regards,

*Kayle Maclou*
*Software Developer*
*Mobile:  **+27 (0)62 478 3284*
*Teams:  ***@***.*** ***@***.***>*
*Zoom:   ***@***.*** ***@***.***>*
*Profile:  **https://www.linkedin.com/in/kayle-maclou/
<https://www.linkedin.com/in/kayle-maclou/>*
